src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -2255,7 +2273,13 @@ static S1 MayWrap(in Span<int> arg)
         }
     }
 ";
-            CreateCompilationWithMscorlibAndSpan(text, parseOptions: TestOptions.Regular.WithLanguageVersion(languageVersion)).VerifyDiagnostics();
+            CreateCompilationWithMscorlibAndSpan(text, parseOptions: TestOptions.Regular.WithLanguageVersion(languageVersion)).VerifyDiagnostics(
+                // (25,20): error CS8347: Cannot use a result of 'Program.MayWrap(in Span<int>)' in this context because it may expose variables referenced by parameter 'arg' outside of their declaration scope
+                //             arg1 = MayWrap(arg2);
+                Diagnostic(ErrorCode.ERR_EscapeCall, "MayWrap(arg2)").WithArguments("Program.MayWrap(in System.Span<int>)", "arg").WithLocation(25, 20),
```

Comment:
>ErrorCode.ERR_EscapeCall [](http://example.com/codeflow?start=27&length=24)

Did `-langversion:10` behavior change?

Reply:
No, I messed up here.

Reply:
It looks like the `-langversion:10` behavior has still changed. Is the test currently failing?